www/heatmap.html

```diff
@@ -0,0 +1,239 @@
+        </div>
+
+        <div id="video" class="container pad">
+            <hr class="featurette-divider">
+            <h2 class="text-center" style="margin-bottom:0px">Live Demo</h2><br><p class="text-center padright" style="font-size:20px">This demo does not work with an ad-blocker.</p>
+            <div id="heatmapContainer" style="margin: auto; width: 100%; height: auto; max-width: 700px">
+                    <img id="webgazer-pic" src="./media/heatmap-demo.png" style="width: 100%; height: auto; max-width: 700px">
+            </div>
+            <script src="./js/heatmap.js"></script>
+            <script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
+            <script src="./js/heatmap-data.js"></script>
+            <script src="./js/heatmap-webgazer.js"></script>
+       </div>
+
+        <div id="heatmap-usage" class="container pad">
+            <hr class="featurette-divider">
+            <h2 class="text-center">Usage</h2>
+            <br>Begin by creating a div element that will house all of the heatmap's visual rendering on your page.
+            
+<pre><code class="html">&lt;!-- Heatmap Container --&gt;
+&lt;div id="heatmapContainer">&lt;div&gt;</code></pre>
+
+            <p><br>Create HTML script elements that import the javascript files needed for the visualization. The script element needs to be given the location that the <code class="javascript">heatmap.js</code> file is stored. <b>Note:</b> The period in <code class="javascript">"./js/heatmap.js"</code> represents the current directory.</p>
+
+<pre><code class="html">&lt;!-- Script Element --&gt;
+&lt;script src="./js/heatmap.js"&gt;&lt;script&gt; 
+</code></pre>
+
+<p><br>Initialize a heatmap instance with a few lines of javascript.</p>
+<pre><code class="javascript hljs">var heatmap = h337.create({
+    container: document.getElementById('heatmapContainer') // refers to the container that you created above
+});
+</code></pre>
+
+            <p><code class="javascript">Heatmap.js</code> supports many kinds of visual configurations that change the heatmaps appearance. View these settings in the initialization <a href="https://www.patrick-wied.at">documentation.</a><p>
+            <p><br>Calling the <code class="javascript">setData()</code> function on the <code class="javascript">heatmap</code> variable: <code class="javascript">heatmap.setData(data)</code> allows the user to provide the heatmap instance with data to display. The data has to be in JSON format, below is an example of how the information should be structured.</p>
+
+<pre><code class="javascript hljs">{
```

Comment:
Is this necessary when it's repeated below just with `var heatmap_data.html` at the front?

Reply:
I think it might be better to keep it for users who many not be familiar with JSON format. I wanted to give them an example of a couple steps one might take to turn the data they have locally into something they could import easily.